### ISSUE TITLE
[FR] Skip Items Option When Scrolling

### ISSUE THREAD

**Issue by Quest79** [p455w0rd/DankNull#288]
This mod is great. What isnt great is scrolling through 30+ blocks to find the one I want. There is an option to open the GUI and manually select the block, however I feel that most items I store in the dank/null are blocks I never need to actively select. For those blocks it would be nice to have an option to skip them when scrolling through (shift+mouse-wheel). 

Alternatively it would be beneficial to have an option for 'favorite blocks' with a separate shortcut to scroll through only the favorites (alt+mouse-wheel for example) 

Thanks.

**Comment by cjm721** [comment]
Would think it be simpler to just have two dank nulls, although I do see the QoL. Will think about it a bit more

**Comment by CurlyFlow** [comment]
Maybe just allow to drag the items inside the dank null, so we could arrange the buildings blocks together...?

**Comment by Quest79** [comment]
I personally feel like having two dank nulls would start to erode the core point of the mod; tons of blocks/items out of my inventory in a sweet central area. I appreciate your consideration.

**Comment by CurlyFlow** [comment]
Yeah, thats why i it would be cool if you could rearrange the items in the dank null, so its better to make your building blocks near by, good to reach via scroll wheel.... (i dont know how i should move 500k cobblestone...)
